app/components/admin/participants/table.html.erb

```diff
@@ -17,4 +17,4 @@
   </table>
 </div>
 
-<%== govuk_paginate profiles, @pagy %>
+<%== govuk_pagination(pagy: @pagy) %>
```

Comment:
something I noticed about pagination is that it's no longer centered within the page, is this intentional to adhere to the component version?